src/main/java/io/zeebe/monitor/zeebe/importers/JobImporter.java

```diff
@@ -2,29 +2,37 @@
 
 import io.zeebe.exporter.proto.Schema;
 import io.zeebe.monitor.entity.JobEntity;
+import io.zeebe.monitor.entity.ProcessInstanceEntity;
 import io.zeebe.monitor.repository.JobRepository;
+import io.zeebe.monitor.repository.ProcessInstanceRepository;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
 @Component
 public class JobImporter {
 
   @Autowired private JobRepository jobRepository;
+  @Autowired private ProcessInstanceRepository processInstanceRepository;
 
   public void importJob(final Schema.JobRecord record) {
 
     final String intent = record.getMetadata().getIntent();
     final long key = record.getMetadata().getKey();
     final long timestamp = record.getMetadata().getTimestamp();
-
+    final ProcessInstanceEntity processInstance = processInstanceRepository.findById(record.getProcessInstanceKey())
+            .orElseGet(() -> {
```

Comment:
all others throw the exception and only this importer creates a default process instance. Why?